docs/articles/documentation/guides/advanced-guides/test-https-features-and-http2-websites.md

```diff
@@ -61,6 +137,16 @@ await runner
     .run();
 ```
 
+### Use a Trusted Certificate
+
+You can use a certificate signed by a trusted authority to launch tests with TestCafe. We recommend that you prefer a trusted certificate over self-signed if your security policy permits. The browsers are then able to open the tested page without workarounds.
```

Comment:
If you agree with my above suggestion to move this text before the "Use a trusted certificate" and "Use a self-signed certificate" sections, you can delete it from here.